src/cloudsc_gpu/cloudsc_driver_gpu_scc_field_mod.F90

```diff
@@ -104,68 +104,78 @@ SUBROUTINE CLOUDSC_DRIVER_GPU_SCC_FIELD( &
-    CALL FIELD_STATE%F_PAPH%GET_DEVICE_DATA_RDONLY(PAPH)
-    CALL FIELD_STATE%F_PLSM%GET_DEVICE_DATA_RDONLY(PLSM)
-    CALL FIELD_STATE%F_LDCUM%GET_DEVICE_DATA_RDONLY(LDCUM)
-    CALL FIELD_STATE%F_KTYPE%GET_DEVICE_DATA_RDONLY(KTYPE)
-    CALL FIELD_STATE%FIELDS_RDONLY(13)%PTR%GET_DEVICE_DATA_RDONLY(PLU)
-    CALL FIELD_STATE%FIELDS_RDONLY(14)%PTR%GET_DEVICE_DATA_RDONLY(PSNDE)
-    CALL FIELD_STATE%FIELDS_RDONLY(15)%PTR%GET_DEVICE_DATA_RDONLY(PMFU)
-    CALL FIELD_STATE%FIELDS_RDONLY(16)%PTR%GET_DEVICE_DATA_RDONLY(PMFD)
-    CALL FIELD_STATE%FIELDS_RDONLY(17)%PTR%GET_DEVICE_DATA_RDONLY(PA)
-    CALL FIELD_STATE%F_PCLV%GET_DEVICE_DATA_RDONLY(PCLV)
-    CALL FIELD_STATE%FIELDS_RDONLY(18)%PTR%GET_DEVICE_DATA_RDONLY(PSUPSAT)
-    CALL FIELD_STATE%FIELDS_RDONLY(19)%PTR%GET_DEVICE_DATA_RDONLY(PLCRIT_AER)
-    CALL FIELD_STATE%FIELDS_RDONLY(20)%PTR%GET_DEVICE_DATA_RDONLY(PICRIT_AER)
-    CALL FIELD_STATE%FIELDS_RDONLY(21)%PTR%GET_DEVICE_DATA_RDONLY(PRE_ICE)
-    CALL FIELD_STATE%FIELDS_RDONLY(22)%PTR%GET_DEVICE_DATA_RDONLY(PCCN)
-    CALL FIELD_STATE%FIELDS_RDONLY(23)%PTR%GET_DEVICE_DATA_RDONLY(PNICE)
+    
+    CALL FIELD_STATE%AUX%F_PT%GET_DEVICE_DATA_RDONLY(PT)
+    CALL FIELD_STATE%AUX%F_PQ%GET_DEVICE_DATA_RDONLY(PQ)
+    CALL FIELD_STATE%AUX%F_PVFA%GET_DEVICE_DATA_RDONLY(PVFA)
+    CALL FIELD_STATE%AUX%F_PVFL%GET_DEVICE_DATA_RDONLY(PVFL)
+    CALL FIELD_STATE%AUX%F_PVFI%GET_DEVICE_DATA_RDONLY(PVFI)
+    CALL FIELD_STATE%AUX%F_PDYNA%GET_DEVICE_DATA_RDONLY(PDYNA)
+    CALL FIELD_STATE%AUX%F_PDYNL%GET_DEVICE_DATA_RDONLY(PDYNL)
+    CALL FIELD_STATE%AUX%F_PDYNI%GET_DEVICE_DATA_RDONLY(PDYNI)
+    CALL FIELD_STATE%AUX%F_PHRSW%GET_DEVICE_DATA_RDONLY(PHRSW)
+    CALL FIELD_STATE%AUX%F_PHRLW%GET_DEVICE_DATA_RDONLY(PHRLW)
+    CALL FIELD_STATE%AUX%F_PVERVEL%GET_DEVICE_DATA_RDONLY(PVERVEL)
+    CALL FIELD_STATE%AUX%F_PAP%GET_DEVICE_DATA_RDONLY(PAP)
+    CALL FIELD_STATE%AUX%F_PLU%GET_DEVICE_DATA_RDONLY(PLU)
+    CALL FIELD_STATE%AUX%F_PSNDE%GET_DEVICE_DATA_RDONLY(PSNDE)
+    CALL FIELD_STATE%AUX%F_PMFU%GET_DEVICE_DATA_RDONLY(PMFU)
+    CALL FIELD_STATE%AUX%F_PMFD%GET_DEVICE_DATA_RDONLY(PMFD)
+    CALL FIELD_STATE%AUX%F_PA%GET_DEVICE_DATA_RDONLY(PA)
+    CALL FIELD_STATE%AUX%F_PSUPSAT%GET_DEVICE_DATA_RDONLY(PSUPSAT)
+    CALL FIELD_STATE%AUX%F_PLCRIT_AER%GET_DEVICE_DATA_RDONLY(PLCRIT_AER)
+    CALL FIELD_STATE%AUX%F_PICRIT_AER%GET_DEVICE_DATA_RDONLY(PICRIT_AER)
+    CALL FIELD_STATE%AUX%F_PRE_ICE%GET_DEVICE_DATA_RDONLY(PRE_ICE)
+    CALL FIELD_STATE%AUX%F_PCCN%GET_DEVICE_DATA_RDONLY(PCCN)
+    CALL FIELD_STATE%AUX%F_PNICE%GET_DEVICE_DATA_RDONLY(PNICE)
+    
+    CALL FIELD_STATE%AUX%F_PAPH%GET_DEVICE_DATA_RDONLY(PAPH)
+    CALL FIELD_STATE%AUX%F_PLSM%GET_DEVICE_DATA_RDONLY(PLSM)
+    CALL FIELD_STATE%AUX%F_LDCUM%GET_DEVICE_DATA_RDONLY(LDCUM)
+    CALL FIELD_STATE%AUX%F_KTYPE%GET_DEVICE_DATA_RDONLY(KTYPE)
+    CALL FIELD_STATE%AUX%F_PCLV%GET_DEVICE_DATA_RDONLY(PCLV)
+    
     CALL FIELD_STATE%TENDENCY_TMP%F_T%GET_DEVICE_DATA_RDONLY(TEND_TMP_T)
```

Comment:
Should these not also get data movement calls on the FieldGang if the packed option is enabled, like we have for `FIELD_STATE%AUX` and `FIELD_STATE%FLUX`?

Reply:
Good catch!